hack/run-wmcb-ci-e2e-test.sh

```diff
@@ -3,57 +3,79 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-RELEASE="release-4.6"
-GIT_CLONE="git clone --branch $RELEASE --single-branch"
 SKIP_VM_SETUP=""
 VM_CREDS=""
 
-while getopts ":v:s" opt; do
-  case ${opt} in
-    v ) # process option for providing existing VM credentials
-      VM_CREDS=$OPTARG
-      ;;
-    s ) # process option for skipping setup in VMs
-      SKIP_VM_SETUP="-skipVMSetup"
-      ;;
-    \? )
-      echo "Usage: $0 [-v] [-s]"
-      exit 0
-      ;;
-  esac
-done
-
-# TODO: Add steps to execute code that will kick off the e2e tests in CI
+# downloads the oc binary but we are just using kubectl as we dont need oc specific commands, oc binary depends on kubectl
+# using only kubectl binary reduces the hassle of moving both oc and kubectl to path. We can use kubectl directly from /tmp
```

Comment:
I don't understand the reasoning to use `kubectl`. There is no reason to use `kubectl` just because we don't use `oc` features. I also don't follow the hassle of moving both binaries when only is being used. Moreover can't you use `oc` directly from `/tmp`.

PS: I am fine using `kubectl`. I just don't follow your reasoning for doing so.

Reply:
My bad, my assumption was that we need kubectl binary for oc binary to work. Just checked that oc works perfectly without kubectl as well.